src/meta/src/barrier/command.rs

```diff
@@ -42,7 +44,10 @@ pub enum Command {
     ///
     /// Barriers from all actors marked as `Created` state will be collected.
     /// After the barrier is collected, it does nothing.
-    Plain(Option<Mutation>),
+    Plain {
+        mutation: Option<Mutation>,
+        changed_table_id: ChangedTableState,
+    },
```

Comment:
The `Plain` command should only be used for normal checkpoints and not be used for configuration change. Do we really need this `changed_table_id`?

Reply:
I use a new design like we said.